sql/core/src/test/resources/sql-tests/inputs/charvarchar.sql

```diff
@@ -23,6 +23,7 @@ desc formatted char_tbl1;
 
 alter table char_tbl1 change column c type char(6);
 alter table char_tbl1 change column c type char(5);
+alter table char_tbl1 change column c comment 'text';
```

Comment:
BTW, as SPARK-49843 says `Fix ... char/varchar columns`, could you check the same for the varchar column `v`

Reply:
Sure, fixed